scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
+
+    # run linter with file output for Jenkins
+    npm run lint:ci
```

Comment:
It would be nice to conditionally run this command only when within the Jenkins environment. I believe there's a few ENV variables that only Jenkins sets that could be used to detect that state, but I can't seem to find the documentation for it. Maybe @azavea/operations can point us in the right direction?